packages/components/atoms/f-form-field/test/visual/f-form-field.visual.spec.js

```diff
@@ -0,0 +1,44 @@
+const FormField = require('../../test-utils/component-objects/f-form-field--visual.component');
+
+let formField;
```

Comment:
It is better to scope this within the describe block. You could add it inside the first describe.

Reply:
Ok. I'll add to that now.